spec/enumbler_spec.rb

```diff
@@ -240,7 +242,12 @@ class Sky < ApplicationRecord
   describe '.enumbler_label_column_name' do
     before { Feeling.seed_the_enumbler! }
     it 'adds the label to the correct column' do
-      expect(Feeling.verklempt.emotion).to eq 'overcome with emotion'
+      expect(Feeling.verklempt.emotion).to eq 'overcame with emotion'
+      expect(Feeling.verklempt.label).to eq('blue')
+      expect(Feeling.verklempt.enumble.label).to eq('overcame with emotion')
```

Comment:
Tests look good.  I would say just be consistent with `.to eq 'the thing'`

And reminds me we should upgrade rubocop and add a rubocop step to this gem as well!